sw/host/opentitanlib/src/bootstrap/mod.rs

```diff
@@ -91,6 +91,11 @@ pub struct BootstrapOptions {
         help = "Bootstrap protocol to use"
     )]
     pub protocol: BootstrapProtocol,
+    #[structopt(
+        long,
+        help = "Whether to reset target and clear UART RX buffer after bootstrap. For CW310 only."
+    )]
+    pub no_reset_and_clear_rx: Option<bool>,
```

Comment:
Shall we drop the negative in the name and make the default be to _not_ reset and clear?

Rationale: The user-interactive case is arguably one where we'd like individual commands to be limited in scope by default (so-called "UNIX style"), so the timing and details of what happens can be carefully controlled. In addition, the problem we're trying to solve here is specific to CI and the complexity of the expressions that determine whether a test passes or fails. Finally, arguments ought to be tailored to the most common user-interactive case, since the user must type them out (unlike CI, which has a script prepared).

Then for CI's case, we can make the bazel test runner include the argument.

Does that sound reasonable to you?

Reply:
I'd prefer having this enabled by default because I'd like the output start after the new program runs, i.e. the thing that I'm interested in.

Also, is the problem really specific to CI? I haven't tried this case specifically (I always use `bazel test`) but I'd think the same problem could happen when you use opentitantool on a single test manually.

@cfrantz any preferences?

Reply:
It's specific to CI / bazel: The problem we were solving was that a batch of automated tests were run in bazel and attempting to check whether each test passed or failed, but one test program's output spilled over to another's.

In the general case, we are bootstrapping something that might not even be an OTTF-based test, and we might want to see any UART output related to bootstrapping itself. In other words, with this default, we are losing data for the general case.

That said, this isn't a hill I'd choose to die on. Mostly I wanted to make sure we're making sure to consider the non-automated case when choosing argument defaults ;)

Reply:
Update: Modified to include the bootstrap output.